src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -124,5 +128,17 @@ public static class FunctionMetadataExtensions
 
         private static Uri GetFunctionHref(string functionName, string baseUrl) =>
             new Uri($"{baseUrl}/admin/functions/{functionName}");
+
+        private static Uri GetFunctionInvokeUrlTemplate(string baseUrl, string functionName, IWebJobsRouter router)
+        {
+            var template = router?.GetFunctionRouteTemplate(functionName);
```

Comment:
Updated this to use a new router API that returns the URL template for a function.

Although the API is useful in other contexts, I'm not sure this is one of them. The router is only aware of enabled and loaded functions, so if a function is disabled/excluded, that route would not be available.

Since the APIs exposed by this controller go directly to the metadata to get the functions list, we'll still have information returned for disabled functions, but no URL templates.

For this API, we'll probably need to parse the metadata and load the route from the function configuration, taking the host base route into consideration.

Leaving this as is for now, but we'll need to revisit if this behavior is not desirable (which likely isn't). As is, we'll only have templates returned for enabled functions.

Reply:
This seems problematic - can't we just return a route template for the function regardless of enablement (pure metadata)? We have all the info we need to build that right?

Reply:
Disabled, yes. Excluded (e.g. functions in not in the white list, if one is used) no. But maybe that is OK.

Reply:
Yeah, excluded is ok